spec/features/user_can_submit_posts_spec.rb

```diff
@@ -1,11 +1,19 @@
 require 'rails_helper'
+require_relative '../../app/models/post.rb'
+require 'date'
 
 feature "Timeline" do
   scenario "Can submit posts and view them" do
     visit "/posts"
-    click_link "New post"
-    fill_in "Message", with: "Hello, world!"
-    click_button "Submit"
+    fill_in "post[message]", with: "Hello, world!"
+    click_button "Post"
     expect(page).to have_content("Hello, world!")
+    date = DateTime.now.strftime('%d %b %Y')
```

Comment:
lines 11 & 12 - are they covered by the next test? 
testing date now feels vulnerable to the test falsely passing eg. if the page showed today's date anyway
we suggest that these two lines are removed